frontend/src/metabase/actions/containers/ActionCreator/ActionCreator.tsx

```diff
@@ -94,33 +98,37 @@ function ActionCreator({
 
   const [showSaveModal, setShowSaveModal] = useState(false);
 
-  const isEditable = model.canWriteActions();
+  const isEditable = isNew || model.canWriteActions();
```

Comment:
I'm not sure if it's good or bad that `model` will always be there regardless of whether or not `ActionCreator` is open with the `+ new` button.

When I think about that on one hand, it feels like `model` should be `undefined` since we definitely do not know what model users are going to save the action to. 

But on the other hand, this could be good since it don't break the code that expect to `model` to always be there e.g. when calling `model.id()` (which won't be called anyway in case of opening action creator from `+ new` button.

I don't have any suggestions off the top of my head, but I wish the path `isNew` involves would be more well-sectioned.